data/csp/sap-converged-cloud/sle15/packages.yaml

```diff
@@ -0,0 +1,12 @@
+packages:
+  _namespace_default_bootloader: Null
+  _namespace_default_shim: Null
+  _namespace_openstack_bootloader:
+    package:
+      - _attributes:
+          name: grub2-arm64-efi
+          arch: aarch64
```

Comment:
As we don't have arm we can probably drop this

Reply:
Arm is a configuration in the build service. It cannot be controlled on a flavor basis, all images built with this description will be built for x86_64 and aarch64. Therefore this is needed or we'll have build failures.